docs/Command-line-client.md

```diff
@@ -43,6 +43,19 @@ Syntax: `storm jar topology-jar-path class ...`
 
 Runs the main method of `class` with the specified arguments. The storm jars and configs in `~/.storm` are put on the classpath. The process is configured so that [StormSubmitter](javadocs/org/apache/storm/StormSubmitter.html) will upload the jar at `topology-jar-path` when the topology is submitted.
 
+When you want to ship other jars which is not included to application jar, you can pass them to `--jars` option with comma-separated string.
```

Comment:
It would be ideal if we can show same artifacts with different versions as warning, but I'm not sure it would be possible, especially application jar is an uber jar so we can't get artifacts within that jar.

If you mean we just would be better to show some general warnings, yes that would be good to have. Btw, this is closer to general behavior of determining worker classpath (even without these options), so it would be better to explain this in general manner.
I'm not sure where is the best place to address this. Let me think about it.